pkg/kubelet/rkt/rkt.go

```diff
@@ -190,11 +191,31 @@ func (r *runtime) runCommand(args ...string) ([]string, error) {
 	return strings.Split(strings.TrimSpace(string(output)), "\n"), nil
 }
 
-// makePodServiceFileName constructs the unit file name for a pod using its UID.
-func makePodServiceFileName(uid types.UID) string {
+// makePodServiceFileName constructs the unit file name for a pod using
+// its pod UID and rkt UUID. We use the rkt UUID here because it's not
+// clear how to update and reload the service file once the service remains
+// after exited.
```

Comment:
If I understand correctly, you simply want a unique ID for the file name. Why would you need to update the service file?